src/multicloud-connector/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+1.0.0b1
+++++++
+* Initial release.
```

Comment:
`az arc-multicloud public-cloud-connector list` command

1. Command says `list connector resources by subscriptionId` but there is no argument to provide the subscriptionId. Interestingly there is an argument for `resourcegroup`. This seems odd.
![image](https://github.com/user-attachments/assets/155ac198-46a2-4149-bdb6-43c848989c1c)

2. `-o table` prints weird resource group names like `clitest.rgy7xqoupodpyivrat6cuddih5wgnzd5rey6tir2xophpvkqckipflrawo2wred7tvs`, looks like a formatting issue. can we suppress the global options?

![image](https://github.com/user-attachments/assets/28d43674-5abd-4a6c-a3b9-deb456b76cc0)